types/nuclear-js/tslint.json

```diff
@@ -0,0 +1,8 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "no-unnecessary-type-assertion": false,
```

Comment:
instead of exceptions in tslint.json, please use `tslint:disable-next-line` in the source.

Reply:
👍 done.